### ISSUE TITLE
Multiple GPUs?

### ISSUE THREAD

**Issue by jmrichardson** [NLeSC/mcfly#268]
Hi, I have a server with multiple GPUs?  Is it possible run mcfly on multiple GPUs in parallel for example with:

`mcfly.find_architecture.train_models_on_samples`

I tried running a test and it appears to be running the models consecutively.  Thanks for any help.

**Comment by dafnevk** [comment]
That's right, parallel execution of the models is currently not implemented. It would be a nice feature to have. See also the related issue #167.
